packages/gatsby-link/src/__tests__/index.js

```diff
@@ -1,10 +1,13 @@
 import React from "react"
-import ReactDOM from "react-dom"
+import { render, cleanup } from "react-testing-library"
 import {
   createMemorySource,
   createHistory,
   LocationProvider,
 } from "@reach/router"
+import Link from "../"
+
+afterEach(cleanup)
 
 const getInstance = (props, pathPrefix = ``) => {
```

Comment:
Btw is this still relevant to test?

Reply:
Yeah — it lets tests change the `pathPrefix`. So tests should use it generally. It does need the context removed I'm noticing.

Reply:
Though it looks like it's easier to merge it into what you're doing

Reply:
Okay, so instead of using `import Link from '../'` we should use the `getInstance` if I understand it correctly?

Reply:
Importing is fine. It's just we need to test adding path prefixes.

Reply:
Sorry I'm sure that I'm following, don't we already have test for that in the bottom of the file?